app/experimenter/static/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -68,23 +69,34 @@ const ExperimentDetails: React.FC = () => {
     );
   }
 
-  const launched_statuses = [ExperimentStatus.LIVE, ExperimentStatus.COMPLETE];
+  const buttonsDisabled = experimentData.status !== ExperimentStatus.DRAFT;
+  let buttonsClass = "btn btn-primary";
+  if (buttonsDisabled) {
+    buttonsClass = "btn btn-secondary";
+  }
+
+  const buttonsShown = ![
+    ExperimentStatus.LIVE,
+    ExperimentStatus.COMPLETE,
+  ].includes(experimentData.status);
+
   let changeStatusButtons;
-  if (!launched_statuses.includes(experimentData.status)) {
+  if (buttonsShown) {
     changeStatusButtons = (
       <div className="d-flex mt-4">
         <span>
           <Link
-            className="btn btn-secondary"
-            to={`/${experimentData.slug}/edit/`}
+            className={buttonsClass}
+            to={buttonsDisabled ? "#" : `/${experimentData.slug}/edit/`}
           >
             Back
           </Link>
         </span>
 
         <span className="flex-grow-1 text-right">
           <button
-            className="btn btn-primary"
+            className={buttonsClass}
```

Comment:
Ah, the interesting side effect here is that I _believe_ that the `buttonsClass` is shared for both the "Back" and "Request Approval" buttons (whereas previously the Back was btn-secondary and the "Request Approval" is btn-primary).

Not a deal/merge blocker, but might need a bit of fine tuning in the future.

Reply:
Yeah I talked to Ana about this about making it consistent, ie `blue` is enabled and `grey` is disabled, but I don't think it made its way back into the wireframes yet.

@AnaMedinac

Reply:
Sure, sure, sure... naming issues and UX-crimes aside, that'd work.
Let's merge and unblock this and we can leave it to our future selves to noodle with this.